fmt-tests/basics/buildfile

```diff
@@ -0,0 +1,45 @@
+include ../test-main/
+
+#test_names =
+#  args-test \
+#  assert-test \
+#  base-test
+gtest_test_names = args-test assert-test base-test chrono-test color-test compile-fp-test compile-test format-test gtest-extra-test noexception-test os-test ostream-test printf-test std-test unicode-test xchar-test
+
+# header-only-test <- header only mode seemingly not supported by this build2 package?
```

Comment:
Yes it was not supported, I basically had the same reasons as explained [in the packaging guide](https://build2.org/build2-toolchain/doc/build2-toolchain-packaging.xhtml#dont-header-only) although that wouldnt prevent support of header-only mode later if the demand is there. So far nobody needed it.